### PR TITLE
Open PDF files by default when PDF viewer is disabled

### PR DESCRIPTION
**Pull request by Raudius**
Related to https://github.com/nextcloud/files_pdfviewer/issues/649

### CODE REVIEW

File: lib/Capabilities.php [hunk 1]
```diff
@@ -82,14 +83,17 @@ class Capabilities implements ICapability {
 	private $capabilitiesService;
 	/** @var PermissionManager */
 	private $permissionManager;
+	/** @var AppManager */
+	private $appManager;
 
 	private $capabilities = null;
 
-	public function __construct(IL10N $l10n, AppConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager) {
+	public function __construct(IL10N $l10n, AppConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager, AppManager $appManager) {
```
Comment (juliushaertl) [inline lib/Capabilities.php:91]:
```suggestion
	public function __construct(IL10N $l10n, AppConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager, IAppManager $appManager) {
```

Should use the IAppManager from the OCP namespace which is considered the public and stable API.

**Review by juliushaertl — COMMENTED** [review]
Small comment regarding the app manager, but otherwise LGTM 👍

File: lib/Capabilities.php (hunk 1 above)
Comment (Raudius) [reply lib/Capabilities.php]: Fixed! 

Also modified my include path in PhpStorm to just include the public API, should avoid this in the future :)

**Pull request merged**